awx/api/generics.py

```diff
@@ -472,7 +476,11 @@ def skip_related_name(name):
 
 class ListCreateAPIView(ListAPIView, generics.ListCreateAPIView):
     # Base class for a list view that allows creating new objects.
-    pass
+    def perform_create(self, serializer):
+        super().perform_create(serializer)
+        if serializer.Meta.model in permission_registry.all_registered_models:
+            if self.request and self.request.user:
+                give_creator_permissions(self.request.user, serializer.instance)
```

Comment:
trying to think of other resources that are created in non-conventional ways

ReceptorAddress in the create_or_update method of the InstanceSerializer
- I think this is admin-only anyways so maybe doesn't matter too much

BulkHost and BulkLaunch endpoints

Reply:
Good question. Those bulk endpoints create hosts & jobs. Hosts defer to inventory permission, and jobs defer to job template (or organization as fallback) permission. So no permission action is needed in those cases.